src/autocomplete.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+# -*- coding: utf -8 -*-
+
+from __future__ import unicode_literals
+from trie import Trie
+
+
+class Autocomplete(object):
+    """Define class Autocomplete"""
+    def __init__(self, vocabulary, max_completions=5):
+        """Create an instance of the class."""
+        self._trie = Trie(iterable=vocabulary)
+        self.max_completions = max_completions
+
+    def autocomplete(self, token=None):
+        """Return list of autocomplete suggestions, based on vocabulary."""
+        result = []
+        counter = 0
```

Comment:
Rather than using a counter, it would be a little simpler to just check `len(result)` against `self.max_completions`.